### DISCUSSION TITLE
Organization as default in propriety field

### DISCUSSION THREAD

**Discussion by KBCACheia** [dani-garcia/vaultwarden#2003] [General]
As the title says, is it possible to put Organization as default in propriety field?
I would like to set it because, the main use of my server is organizational, but i want user can put personal stuffs, if they want to.

**Comment by bokkabonga** [comment]
I´m not sure if I understand what you´re trying to ask here. Do you want every user to be added to a "default organisation"? Because i don´t think that this is currently possible

**Reply by KBCACheia** [reply]
> I´m not sure if I understand what you´re trying to ask here. Do you want every user to be added to a "default organisation"? Because i don´t think that this is currently possible

Hi, thanks for the answer,   when i add a item, in the propriety field, where there is "Who own this item?", by default is your e-mail as owner. I would like to set one of my organization by default. However, i notice when i chose the collection first, the organization appears as default. Then, is not a issue anymore.

**Reply by BlackDex** [reply]
That would be a frontend/web-vault item to be added. And is not something we can add on the server-side.
Best place to look for this is here: https://community.bitwarden.com/c/feature-requests/5

**Reply by KBCACheia** [reply]
> That would be a frontend/web-vault item to be added. And is not something we can add on the server-side. Best place to look for this is here: https://community.bitwarden.com/c/feature-requests/5

Thanks, i imagined that. But by now, is not a big issue. I just click on collection and the organization is the default option.
